sds_data_manager/stacks/efs_lambda_stack.py

```diff
@@ -0,0 +1,147 @@
+from aws_cdk import (
+    Duration,
+    Fn,
+    Stack,
+    aws_iam,
+    aws_lambda,
+)
+from aws_cdk import (
+    aws_ec2 as ec2,
+)
+from aws_cdk import (
+    aws_events as events,
+)
+from aws_cdk import (
+    aws_events_targets as targets,
+)
+from aws_cdk import (
+    aws_s3 as s3,
+)
+from constructs import Construct
+
+
+class EFSWriteLambda(Stack):
+    """
+    Creates some Lambdas that write to the EFS file system.
+    """
+
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        vpc: ec2.Vpc,
+        data_bucket: s3.Bucket,
+        **kwargs,
+    ) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # Create a role for the EFS Lambda
+        # Grant the Lambda identity role access to the VPC/EFS
+        iam_role_name = "efs-lambda-role"
+        efs_lambda_role = aws_iam.Role(
+            self,
+            iam_role_name,
+            role_name=iam_role_name,
+            managed_policies=[
+                aws_iam.ManagedPolicy.from_aws_managed_policy_name(
+                    "service-role/AWSLambdaBasicExecutionRole"
+                ),
+                aws_iam.ManagedPolicy.from_aws_managed_policy_name(
+                    "service-role/AWSLambdaVPCAccessExecutionRole"
+                ),
+                aws_iam.ManagedPolicy.from_aws_managed_policy_name(
+                    "AmazonElasticFileSystemClientReadWriteAccess"
+                ),
```

Comment:
Is this already handled by the below `FullAccess` policy?

Reply:
It may be covered by below. I will test it out.

Reply:
we can remove it.